### ISSUE TITLE
Possible issue when reporting clips?

### ISSUE THREAD

**Issue by reymarkus** [HolodexNet/Holodex#346]
So, I tried reporting a clip, and it seems like it fails to submit the report for some reason whether you are logged in or not. It seems that there might be an issue with your Discord webhooks.

**Repro:**
* Open Holodex, then go to Clips tab
* Pick a clip that you want to report
* Fill out the fields, and submit

**Expected behavior:** Report should go through

**Actual behavior:** Report does not go through, and an error is displayed in the report form.

**Screenshots:**

Report form
![image](https://user-images.githubusercontent.com/46585888/127724729-93068d0b-21cb-4147-9663-498b47c292ab.png)

DevTools network logs
![image](https://user-images.githubusercontent.com/46585888/127724906-d8e34fcb-0ce7-41b3-a3d3-767b46b51036.png)

Discord webhook request and response for reporting content:
```json
{
  "content": "Look what the dog dragged in...",
  "embeds": [
    {
      "title": "Video Report",
      "color": 13840175,
      "fields": [
        {
          "name": "Reason",
          "value": "Video tagged incorrectly"
        },
        {
          "name": "Comments",
          "value": "Incorrectly tagged due to automatic tagging. Sodafunk is not a Holomem and should not be in the list of Hololive-related clips.\n\nAlso check other Sodafunk-focused clips on Hololive-related clips."
        },
        {
          "name": "Video Title",
          "value": "SodaFunk Forgot To Mute Her Mic And Gets Embarrassed~"
        },
        {
          "name": "Channel",
          "value": "Whatopia / Hololive Clipper"
        },
        {
          "name": "Link",
          "value": "https://holodex.net/watch/hvKjvM7Tzis"
        }
      ]
    }
  ]
}
```

```json
{"message": "Unknown Webhook", "code": 10015}
```

**Current browser settings:** Firefox 90, with uBlock Origin enabled (also tested with it disabled), NoScript

**Comment by sphinxrave** [comment]
yeahhh we broke something random after i tripped over a critical system file the other day.

**Comment by reymarkus** [comment]
Looks like it will be resolved in #347, will wait for the update to go live

**Comment by RiceCakess** [comment]
Update is live
